sdk/hdinsight/arm-hdinsight/README.md

```diff
@@ -1,94 +1,112 @@
 ## Azure HDInsightManagementClient SDK for JavaScript
 
-This package contains an isomorphic SDK for HDInsightManagementClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for HDInsightManagementClient.
```

Comment:
Hmm.. This should have been `Node.js` and not `node.js`

@dw511214992 Did you use the script or the code generator?
@deyaaeldeen We should fix this

Reply:
It is in the script: https://github.com/Azure/azure-sdk-for-js/blob/1fab5f538148786cb94d862cb0b56d52afdddea5/common/tools/add-identity-mgmt-lib.js#L58. I will open a PR with the fix shortly.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/15566